cpp/src/community/detail/refine_impl.cuh

```diff
@@ -100,9 +101,9 @@ struct leiden_key_aggregated_edge_op_t {
       if ((louvain_of_dst_leiden_cluster == src_louvain_cluster) &&
           is_dst_leiden_cluster_well_connected) {
         mod_gain = aggregated_weight_to_neighboring_leiden_cluster -
-                   resolution * src_weighted_deg * (dst_leiden_volume - src_weighted_deg) /
-                     total_edge_weight;
-
+                   resolution * src_weighted_deg * dst_leiden_volume / total_edge_weight;
+// NOTE: Disable random moves in refinement phase for now.
+#if 0
```

Comment:
Just to clarify... we've talked about a subsequent PR making a parameter that controls this.

Reply:
That is correct.